src/app.ts

```diff
@@ -13,14 +13,91 @@ interface TemperatureSummary {
   average: number
 }
 
-export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+// Interface
+interface RecordedCity {
```

Comment:
extend from TemperatureReading and Omit temperature property instead of defining the properties manually